packages/dashboard/src/components/appbar.tsx

```diff
@@ -520,7 +520,8 @@ export const AppBar = React.memo(({ extraToolbarItems }: AppBarProps): React.Rea
                 </MenuItem>
                 <Divider />
                 <MenuItem
-                  disabled={!profile.user.is_admin}
+                  // FIXME: reinstate after authz is implemented properly
+                  // disabled={!profile.user.is_admin}
```

Comment:
I think we should keep this check, there should be no reason for non admin to see this tab, even if the concept of "admin" has no meaning now.